Dockerfile

```diff
@@ -25,7 +22,7 @@ COPY --chown=airflow:airflow ${GIT_REPO_DIR} ./${GIT_REPO_DIR}
 
 RUN chmod +x install_dag_in_docker.sh
 
-RUN ./install_dag_in_docker.sh
+#RUN ./install_dag_in_docker.sh
```

Comment:
perhaps need to enable that line again :-)

Reply:
added :)